plugin/src/main/java/com/denizenscript/denizen/utilities/blocks/CuboidBlockSet.java

```diff
@@ -358,14 +366,53 @@ public void rotateOne() {
         blocks = bd;
     }
 
+    public void flipEntitiesX() {
+        if (entities == null) {
+            return;
+        }
+        ListTag outEntities = new ListTag();
+        for (MapTag data : entities.filter(MapTag.class, CoreUtilities.noDebugContext)) {
+            EntityTag entity = data.getObjectAs("entity", EntityTag.class, CoreUtilities.noDebugContext);

```

Comment:
This looks like you mistakenly referenced `pasteEntities` rather than `rotateEntities`? This code is generating EntityTag instances when it should not be

Reply:
When I wrote this I was having some trouble with getting the proper face to be set during the flip if the face was also rotated during pasteEntities. So my original attempt tried to adjust the blockFace used for rotation on the entity, and save it back to the schematic without adjusting the tracked rotation change to prevent pasteEntities from rotating based on the flip. Not the cleanest, and seems like it probably wasn't right...

I reworked this to add the needed Yaw change to the tracked rotation change of the entity, though I'm seeing incorrect facing armor stands from what is expected, and struggling a bit. I'll take another fresh look at it in a couple days if an obvious mistake doesn't pop out. ab8af60022dc9f16118b9fb2f1429cf94882389f